code/ZAS/Airflow.dm

```diff
@@ -43,7 +43,7 @@ atom/movable/GotoAirflowDest(n)
 /mob/proc/airflow_stun()
 	if(isDead() || (flags & INVULNERABLE) || (status_flags & GODMODE))
 		return FALSE
-	if(world.time < last_airflow_stun + zas_settings.Get(/datum/ZAS_Setting/airflow_stun_cooldown))
+	if(world.time < (last_airflow_stun + zas_settings.Get(/datum/ZAS_Setting/airflow_stun_cooldown) + 50))
```

Comment:
magic 50

Reply:
Seconded, explain the meaning of this 50. Explain it in the code.

Reply:
EXPLAIN THIS MADNESS SO WE CAN END THIS

Reply:
It's the knockdown. I didn't change the 5 so I just added a 50. To account for the knockdown time.